callstack/react-native-paper
# fix: improve screen reader experience by fixing redundant FAB.Group item announcements.

## Motivation
The goal is to enhance the screen reader experience by resolving redundant item announcements in the FAB.Group.

## Related Issue
Fixes #4063

## Test Plan
1. **Check FAB.Group Presence:**
   - Verify the presence of FAB.Group in FABExample.tsx.

2. **Android Testing (TalkBack):**
   - Use TalkBack on Android to navigate the FAB.Group.
   - Confirm that each item is announced only once.
   - Ensure that no new accessibility issues are introduced.

3. **iOS Testing (VoiceOver):**
   - Verify the presence and functionality of FAB.Group on iOS devices.
   - Use VoiceOver to navigate the FAB.Group.
   - Confirm that each item is announced only once.
   - Ensure that no new accessibility issues are introduced.


Remember to document any findings, changes, or considerations during the testing process. This comprehensive plan should help ensure a thorough evaluation of the accessibility improvements across different platforms and devices.


## Result

| Android  | IOS |
| ------------- | ------------- |
| <video title src="https://github.com/callstack/react-native-paper/assets/134603758/066c6c33-3064-4a7e-8d8e-80ae33955436">  | <video src="https://github.com/callstack/react-native-paper/assets/134603758/03c51cf7-71e1-406b-9628-a7029d034615">|
